compiler/luci/pass/src/QuantizeDequantizeWeightsPass.cpp

```diff
@@ -369,21 +369,15 @@ struct QuantizeDequantizeWeights final : public luci::CircleNodeMutableVisitor<b
     assert(output_type == loco::DataType::U8 || output_type == loco::DataType::S16);
     LOGGER(l);
     INFO(l) << "QuantizeDequantizeWeights visit node: " << node->name() << std::endl;
-    auto arity = node->arity();
-    for (uint32_t i = 0; i < arity; i++)
-    {
-      auto input_node = node->arg(i);
-      auto circle_node = loco::must_cast<luci::CircleNode *>(input_node);
 
-      if (not is_quantizable(input_node))
-        continue;
+    if (not is_quantizable(node))
+      return false;
```

Comment:
not a scope of this PR but
all return statements in this method is `return false` and there is no `return true`.
Would `void visit(...)` be right signature?

Reply:
> Would void visit(...) be right signature?

Yes. It will be better to use void visitor, as we don't use return values (same for `QuantizeWithMinMaxPass`).